codegen/access_assets.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+def ensight_api_test_assests():
+    """Read ensight_api_test_assets.txt and return it as string.
+
+    Returns
+    -------
+    str
+        The file contents as string.
+    """
+    root = os.path.dirname(os.path.dirname(__file__))
+    assets_file_name = os.path.join(root, "assets", "ensight_api_test_assets.txt")
```

Comment:
Assume the file qould be in its existing location... or in an assets folder inside stc/ansys/api/pyensight. Whatever you prefer